### PR TITLE
Fixes #205 Remove dev-ops module having maven script utils and unused SecureRandomProvider

### LINKED ISSUE #205: Consider removing `devops` module
It's unclear if the `dev-ops` module is adding any value. Maven-scripting is typically done through alias commands, and the DeterministicSecureRandomProvider doesn't appear to be used in any tests at present.

### PR DESCRIPTION
**Pull request by sudheesh001**
Signed-off-by: sudheesh001 <sudheesh1995@outlook.com>

### CODE REVIEW

**Comment by sappenin** [comment]
@adrianhopebailie Any opinion around this PR? 

The `dev-ops` module isn't adding any value IMHO, although I'm curious if you think we should keep `DeterministicSecureRandomAlgorithm.java` around somewhere, such as in `ilp-core` or somewhere else. That class is not used in _this_ project, but I can't remember if you were finding it useful somewhere downstream.

**Comment by sappenin** [comment]
One more point of context - this class doesn't appear to be used anywhere in Github except for forks of `ilp-core`: https://github.com/search?q=DeterministicSecureRandomAlgorithm&type=Code

**Comment by adrianhopebailie** [comment]
`DeterministicSecureRandomAlgorithm` was used to test things that use a PRNG. It allows you to specify the "random" bytes that will be generated so you can write unit tests against it.

**Comment by sappenin** [comment]
@adrianhopebailie Was that ever used anywhere though? Maybe when we had crypto-conditions in the project?

File: dev-ops/util/README.md [hunk 1]
```diff
@@ -1,26 +0,0 @@
-Collection of utility scripts wrapping common task (mvn local install, test,...)
-
```
Comment (sappenin) [inline dev-ops/util/README.md]: I think we need to update the project's README file as well.

**Comment by sudheesh001** [comment]
Looking into the history of the file, it looks like the file was originally in the `ilp-core/src/test/` and later migrated to `dev-ops/src/main`. I don't find any test cases explicitly using this file. I suggest we move it back to the `ilp-core/src/test` in case we'd like to maintain this as a test util for future tests. What do you think @adrianhopebailie @sappenin ?

**Comment by sappenin** [comment]
> I suggest we move it back to the ilp-core/src/test in case we'd like to maintain this as a test util for future tests

I spoke to @adrianhopebailie and we agree - since it's unused, let's remove it. We can always dig it up from the Git history if we need it.

**Comment by sudheesh001** [comment]
Sounds great. I also realized that with this change there was a need to move the `checkstyle.xml` which was previously inside `dev-ops` to the parents to ensure `mvn checkstyle:checkstyle` worked as expected. Updated the PR for the same.

**Pull request merged**